### PR TITLE
Second hw 

### PR DESCRIPTION
**Pull request by BelitskyiAlexandr**
checkstyle problems

### CODE REVIEW

**Comment by BelitskyiAlexandr** [comment]
Could you help me with checkstyle problems in autogen mapper.impl class?

File: pom.xml [hunk 1]
```diff
@@ -99,6 +100,31 @@
 					<linkXRef>false</linkXRef>
```
Comment (Elena-Bruyako) [inline pom.xml:100]:
```suggestion
					<linkXRef>false</linkXRef>
					<sourceDirectories>src</sourceDirectories>
```

**Review by Elena-Bruyako — CHANGES_REQUESTED** [review]
see comments

**Review by Elena-Bruyako — APPROVED** [review]
well done

**Pull request merged**